src/PriorityButtons.js

```diff
@@ -3,20 +3,44 @@ import React from 'react';
 
 function PriorityButtons(props) {
     let displayList = ["high", "medium", "low"];
-    return <span className="priorityButtons">
+
+    console.log("priority is " + props.priority.toString())
+    console.log(props.priorityLevels)
+    // console.log(displayList[indexOf(props.priority)])
+    console.log(displayList[props.priorityLevels.indexOf(props.priority)])
+
+    return <span>
+        <span className="priorityButtons1">
+    <select
+        className={"select"}
+        value={props.priority}
+        // id={props.priority}
+        onChange={props.selectPriority}
+        aria-label={"Hit space to enter priorities, use cursor to select priority, hit space to confirm"}
```

Comment:
Again, don't tell them how to do it. Just tell them what this select field contains (priorities).